src/foundry/client/pixi/webgl/shaders/base.d.ts

```diff
@@ -0,0 +1,669 @@
+export {};
+
+/** @remarks Class name adjusted to avoid name collision with function */
+declare class BaseShaderMixinClass {
+  /**
+   * Common attributes for vertex shaders.
+   * @defaultValue
+   * ```js
+   * `attribute vec2 aVertexPosition;
+   * attribute float aDepthValue;`
```

Comment:
Looks a bit odd at first glance but I guess it _is_ a multiline string, No action needed.